src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -15,11 +15,12 @@
 import {timestampToString} from '../../timestamps.js';
 import DiscussionComment from './discussion-comment.js';
 import DiscussionManager from './discussion-manager.js';
-import {COMMENT_TYPE_QUESTION} from './discussion.js';
 
 export const ELEMENT_DISCUSSION =
     document.querySelector('#discussion-comments');
 
+const COMMENT_TYPE_QUESTION = 'QUESTION';
```

Comment:
Why was this moved?

Reply:
Two of them aren't being used, figured it would make more sense to just have one of them here.

Reply:
Moved them back to discusssion.js

Reply:
ah. yeah they're being used in master now